examples/bigquery-sink/main.tf

```diff
@@ -0,0 +1,15 @@
+variable "api_token" {}
+module "bigquery-sink" {
+  # This is for local reference only; if you're using this module as a published
+  # module from GitHub, the 'source' parameter must refer to it's public location.
+  #  source                      = "github.com/entur/terraform-aiven-kafka-connect/modules/bigquery-sink?ref=vVERSION"
+  source                      = "../../modules/bigquery-sink"
+  aiven_api_token             = var.api_token
+  aiven_project_name          = "ent-sbx"
```

Comment:
hard-coded project - refactor into a variable with "ent-sbx" as the default value?

Reply:
This is an example configuration file depicting usage of the module. We could use more generic strings here.

Reply:
ah, I missed the `examples/`! 
then it makes sense with the current example